packages/react-components/src/templates/ProfileOutputs.tsx

```diff
@@ -0,0 +1,102 @@
+import { ResearchOutputPublishingEntities } from '@asap-hub/model';
+import { css } from '@emotion/react';
+import React, { ComponentProps } from 'react';
+import { SharedResearchList } from '.';
+import { LinkConditional } from '../atoms';
+import { createMailTo } from '../mail';
+import { perRem } from '../pixels';
+import NoOutputsPage from './NoOutputsPage';
+
+const containerStyles = css({
+  padding: `${36 / perRem}em 0`,
+});
+
+export type ProfileOutputsProps = Omit<
+  ComponentProps<typeof SharedResearchList>,
+  'children'
+> & {
+  userAssociationMember: boolean;
+  hasOutputs: boolean;
+  contactEmail?: string;
+  publishingEntity: ResearchOutputPublishingEntities;
+};
+
+const ProfileOutputs: React.FC<ProfileOutputsProps> = ({
+  researchOutputs,
+  numberOfItems,
+  numberOfPages,
+  currentPageIndex,
+  renderPageHref,
+  isListView,
+  cardViewHref,
+  exportResults,
+  listViewHref,
+  hasOutputs,
+  userAssociationMember,
+  contactEmail,
+  publishingEntity,
+}) => (
+  <div css={containerStyles}>
+    {hasOutputs ? (
```

Comment:
So we have the number of items below how does this differ from hasOutputs?

Reply:
I've mostly refactored TeamProfileOutputs to fit the needs of WorkingGroupProfileOutputs, this wasn't added by me and I was unsure how far down the refactoring hole I should go.